content/ember-6-0-released.md

```diff
@@ -0,0 +1,231 @@
+---
+title: Ember 6.0 Released
+image:
+imageMeta:
+  attribution:
+  attributionLink:
+featured: true
+authors:
+  - jared-galanis
+date: 2024-11-20T00:00:00.000Z
+tags:
+  - releases
+  - '2024'
+  - version-6-x
+---
+
+EmberJS 6.0 embodies the commitment to stability and reliability that developers have come to expect from the Ember ecosystem. The Ember team has worked hard to remove all the deprecations and make us move towards Polaris. Get up-to-date with the latest in Ember and enjoy!
```

Comment:
We actually have not removed the deprecations -- we now have them throw in the major so the APIs are removed but the warnings still are there, for at least one release